code/lab/dialogs/render_options.cpp

```diff
@@ -289,15 +294,20 @@ void RenderOptions::open(Button* /*caller*/) {
 	cbp = (Checkbox*)dialogWindow->AddChild(new Checkbox("Render with emissive lighting", 2, y, set_emissive_flag));
 	y += cbp->GetHeight() + 2;
 
-	auto ambient_sldr = new Slider("Ambient Factor", 0, 128, 0, y + 2, set_ambient_factor, dialogWindow->GetWidth());
-	ambient_sldr->SetSliderValue((float)Cmdline_ambient_factor);
+	auto light_sldr = new Slider("Light Brightness", 0.0f, 10.0f, 0, y + 2, set_light_factor, dialogWindow->GetWidth());
+	light_sldr->SetSliderValue(lighting_profile::lab_get_light());
+	dialogWindow->AddChild(light_sldr);
+	y += light_sldr->GetHeight() + 2;
+
+	auto ambient_sldr = new Slider("Ambient Factor", 0.0f, 10.0f, 0, y + 2, set_ambient_factor, dialogWindow->GetWidth());
+	ambient_sldr->SetSliderValue(lighting_profile::lab_get_ambient());
 	dialogWindow->AddChild(ambient_sldr);
 	y += ambient_sldr->GetHeight() + 2;
 
-	auto direct_sldr = new Slider("Direct. Lights", 0.0f, 2.0f, 0, y + 2, set_static_light_factor, dialogWindow->GetWidth());
-	direct_sldr->SetSliderValue(static_light_factor);
-	dialogWindow->AddChild(direct_sldr);
-	y += direct_sldr->GetHeight() + 2;
+	auto emissive_sldr = new Slider("Emissive Amount", 0, 10.0f, 0, y + 2, set_emissive, dialogWindow->GetWidth());
```

Comment:
Amount? Is this a factor? Is this absolute brightness? Can this be given a more concise name, or is this a somewhat arbitrary value?

Reply:
Emissive is a flat amount of white light added to the scene. It's not a factor, in fact it acts more like the horribly named 'ambient factor' currently does. To me it's a pretty arbitrary name, but I'm not very invested in the name either way so if something else is more intuitive to other people then, eh.

Reply:
Okay.
In that case, it's fine as is